app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -222,6 +233,14 @@ const MeetingEditor = () => {
 
   const title = isEditPage ? `Redigerer: ${meeting.title}` : 'Nytt møte';
 
+  const loadReportFromLocalStorage = (form) => {
+    const storedReport = localStorage.getItem(`meeting-${meeting.id}-report`);
+    if (storedReport) {
+      form.change('report', storedReport);
+      setFormKey(formKey + 1);
+    }`meeting-${meeting.id}-report`
```

Comment:
```suggestion
    }
```
Is this supposed to be there?